car_width: 0.2532
min_obstacle_threshould: 2.0
max_obstacle_threshould: 4.0
scan_topic: "/scan"
drive_topic: "/nav"
start_angle: 90 #field of view to avoid in degree (0 to 360)
end_angle: 270 #field of view to aoid in degree (0 to 360)
avoidance_method: 1 # 0->avoid bubble, 1->avoid using disparity
debug: false
debug_scan: "/debug_scan"
debug_marker: "/visualization_marker"
debug_gap: "/gap_visualization"
detect_disparity_threshould: 0.8
max_speed: 6.0
min_gap_size: 7 #increase this if your robot is going too close to the obstacles